src/components/pages/ReservationForm.jsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable react/forbid-prop-types */
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import { createReservations } from '../../redux/reservation/reservationSlice';
+import './reservationForm.css';
+import SidebarComponents from '../sideBar/sideBarComponent';
+
+function ReservationForm({ car }) {
```

Comment:
- [ ] For some reason, I can't see my reservations:

![image](https://github.com/Chandan-devs-tech/drive-easy-frontend/assets/57421823/0d4904f7-cac0-433c-b42b-4503321f0737)

However, if I change to the mobile view, it works:

![image](https://github.com/Chandan-devs-tech/drive-easy-frontend/assets/57421823/87a5695a-4709-42b0-bc66-4f5df4aa7b6b)

Kindly, fix this bug so your app works as intended. 👍